ts/features/lollipop/saga/index.ts

```diff
@@ -61,6 +73,37 @@ export function* deleteCurrentLollipopKeyAndGenerateNewKeyTag() {
   yield* put(lollipopKeyTagSave({ keyTag: newKeyTag }));
 }
 
+export function* lollipopKeyCheckWithServer() {
+  const maybeSessionInformation: ReturnType<typeof sessionInfoSelector> =
+    yield* select(sessionInfoSelector);
+
+  const publicKey: ReturnType<typeof lollipopPublicKeySelector> = yield* select(
+    lollipopPublicKeySelector
+  );
+
+  if (O.isSome(maybeSessionInformation)) {
+    const lollipopAssertionRef =
+      maybeSessionInformation.value.lollipopAssertionRef;
+
+    // eslint-disable-next-line functional/no-let
+    let localAssertionRef;
+
+    if (O.isSome(publicKey)) {
+      const converted = jwkThumbprintByEncoding(
```

Comment:
Is `converted` delivering all the necessary information? Would not something like `publicKeyThumbprint` be better?